<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Unsplash_GUIClass</class>
 <widget class="QMainWindow" name="Unsplash_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unsplash_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="refreshLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Refresh interval (hour)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>77</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="intervalComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>5 min</string>
          </property>
          <item>
           <property name="text">
            <string>5 min</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10 min</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15 min</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30 min</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 day</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="resLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Wallpaper resolution</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>158</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="resComboBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Auto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1280x720</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1280x800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1440x810</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1440x900</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1920x1080</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1920x1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2560x1440</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2560x1600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3440x1440</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3840x2160</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="saveLabel">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Folder for saved wallpapers</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="defaultSaveButton">
            <property name="text">
             <string>Default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="changeFolderButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="saveFolderDisp">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>78</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="aboutButton">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh now</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exitButton">
          <property name="text">
           <string>Hide</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="unsplash_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>aboutButton</sender>
   <signal>clicked()</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_About_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_Refresh_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_Res_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intervalComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_Interval_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_defaultSave_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_changeSave_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>Unsplash_GUIClass</receiver>
   <slot>on_exit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_About_clicked()</slot>
  <slot>on_Refresh_clicked()</slot>
  <slot>on_Res_changed()</slot>
  <slot>on_Interval_changed()</slot>
  <slot>on_defaultSave_clicked()</slot>
  <slot>on_changeSave_clicked()</slot>
  <slot>on_exit_clicked()</slot>
 </slots>
</ui>
